FirebaseRemoteConfigSwift/Sources/PropertyWrapper/RemoteConfigValueObservable.swift

```diff
@@ -61,13 +55,19 @@ internal class RemoteConfigValueObservable<T: Decodable>: ObservableObject {
     if FirebaseApp.app()?.name != appName {
       return
     }
+    applyConfigValue()
+  }
+
+  private func applyConfigValue() {
     do {
       let configValue: RemoteConfigValue = remoteConfig[key]
-      if configValue.source == .remote {
```

Comment:
ahh i see, so previously the PW value would only change if there was a remote value, and does nothing otherwise